README.md

```diff
@@ -58,6 +58,12 @@ The RoboCup @Home Organizing Committee is always looking for volunteers to assis
 
 Contributions and improvements to the rulebook are always welcome in the form of pull requests.
 
+### License
+
+The content of this project itself is licensed under the Creative Commons Attribution Share Alike 4.0 International license, and the underlying source code used to format and display that content is licensed under the GPL v2 license.
```

Comment:
Add this information in the LICENSE file, and add two more files which the GLPv2 and CC-BY-SA-4.0